sda/cmd/auth/main.go

```diff
@@ -270,6 +275,7 @@ func (auth AuthHandler) elixirLogin(ctx iris.Context) *OIDCData {
 	}
 
 	if auth.Config.ResignJwt {
+		log.Debugf("Resigning token for user %s", idStruct.User)
```

Comment:
Putting the comment here instead.

the line `jwt.SubjectKey:    idStruct.Profile,` below should be `jwt.SubjectKey:    idStruct.User,`

Reply:
fixed in ba8e75f8